src/test/java/org/recap/model/ReportsResponseUT.java

```diff
@@ -18,33 +19,18 @@
     @Test
     public void testReportResponse(){
         ReportsResponse reportsResponse = new ReportsResponse();
-        reportsResponse.setAccessionPrivatePulCount(1);
-        reportsResponse.setAccessionPrivateCulCount(1);
-        reportsResponse.setAccessionPrivateNyplCount(1);
-        reportsResponse.setAccessionPrivatePulCount(1);
-        reportsResponse.setAccessionPrivateCulCount(1);
-        reportsResponse.setAccessionPrivateNyplCount(1);
-        reportsResponse.setAccessionPrivatePulCount(1);
-        reportsResponse.setAccessionPrivateCulCount(1);
-        reportsResponse.setAccessionPrivateNyplCount(1);
-        reportsResponse.setDeaccessionPrivatePulCount(1);
-        reportsResponse.setDeaccessionPrivateCulCount(1);
-        reportsResponse.setDeaccessionPrivateNyplCount(1);
-        reportsResponse.setDeaccessionPrivatePulCount(1);
-        reportsResponse.setDeaccessionPrivateCulCount(1);
-        reportsResponse.setDeaccessionPrivateNyplCount(1);
-        reportsResponse.setDeaccessionPrivatePulCount(1);
-        reportsResponse.setDeaccessionPrivateCulCount(1);
-        reportsResponse.setDeaccessionPrivateNyplCount(1);
-        reportsResponse.setOpenPulCgdCount(1);
-        reportsResponse.setOpenCulCgdCount(1);
-        reportsResponse.setOpenNyplCgdCount(1);
-        reportsResponse.setSharedPulCgdCount(1);
-        reportsResponse.setSharedCulCgdCount(1);
-        reportsResponse.setSharedNyplCgdCount(1);
-        reportsResponse.setPrivatePulCgdCount(1);
-        reportsResponse.setPrivateCulCgdCount(1);
-        reportsResponse.setPrivateNyplCgdCount(1);
+        ReportsInstitutionForm reportsInstitutionForm = new ReportsInstitutionForm();
+        reportsInstitutionForm.setInstitution("PUL");
+        reportsInstitutionForm.setAccessionPrivateCount(1);
+        reportsInstitutionForm.setAccessionPrivateCount(1);
+        reportsInstitutionForm.setAccessionPrivateCount(1);
+        reportsInstitutionForm.setDeaccessionPrivateCount(1);
+        reportsInstitutionForm.setDeaccessionPrivateCount(1);
+        reportsInstitutionForm.setDeaccessionPrivateCount(1);
+        reportsInstitutionForm.setOpenCgdCount(1);
```

Comment:
@rajeshbabuk  - Why are we setting the Accession and Deacession count multiple times?

Reply:
It is incorrectly coded. Updated and pushed it Rama.